### PR TITLE
Additional icons

### LINKED ISSUE #355: Please add lichess icon
[Lichess](https://lichess.org) is hosted [here on GitHub](https://github.com/ornicar/lila). Its logo is **already available in SVG** and has a minified version too.

It's worth nothing that Lichess doesn't actually display this knight as the logo on their site but it's shown on embeds, as the favicon on browsers and notably its official [mobile app](https://play.google.com/store/apps/details?id=org.lichess.mobileapp).

You can take your pick from these two for an icon:
1. The [favicon](https://github.com/ornicar/lila/blob/master/public/favicon.svg), which is identical to the [Wikimedia Commons piece](https://commons.wikimedia.org/wiki/File:Chess_nlt45.svg) that it's taken from. Not minified and looks to have been written by hand.
2. The [optimised SVG](https://github.com/ornicar/lila/blob/master/public/piece/cburnett/wN.svg) used in play on the website. It may have been modified to look better on different boards, I'm aware some piece sets have been. If there's any difference, it's indistinguishable.

If you're concerned, the icon is [licensed](https://github.com/ornicar/lila/blob/master/COPYING.md) under the GNU GPLv2+ with credit to [Colin M.L. Burnett](https://en.wikipedia.org/wiki/User:Cburnett).

Thanks in advance.

### LINKED ISSUE #401: Bitmex favricon
Hi there, 

Could you maybe add a bitmex.com favricon? Pretty widley used crypto leverage exchange.

Thank you

### LINKED ISSUE #411: Add Bugcrowd Icon
#### General information

* **App version:** 0.6.3.1-play
* **App source:** Google Play
* **Android Version:** 9
* **Custom ROM:** Samsung OneUI 1.0

#### Request

Add an icon for Bugcrowd, as their accounts [support 2FA](https://docs.bugcrowd.com/docs/two-factor-authentication-1).

### LINKED ISSUE #419: Update Slack thumbnail
https://en.wikipedia.org/wiki/File:Slack_Technologies_Logo.svg

image

### LINKED ISSUE #429: Add airbnb thumbnail
#### General information

* **App version:** 0.6.3.1-play
* **App source:** Google Play
* **Android Version:** 10.0
* **Custom ROM:** AOSP

Can we add airbnb thumbnail as they support 2FA for internal users.

https://upload.wikimedia.org/wikipedia/commons/6/69/Airbnb_Logo_B%C3%A9lo.svg

### LINKED ISSUE #435: Add Fastmail icon
Since `Fastmail` also have 2FA, it would be great to have its icon available in `andOTP` :tada:

Here is the [link](https://www.fastmail.com/images/press/logos/FM-Icon-RGB.png) for the logo.

Thanks in advance!

### LINKED ISSUE #436: Add jetbrains icon
Since they now support 2fa, it would be nice to have their logo available.

https://www.jetbrains.com/company/brand/logos/

### LINKED ISSUE #438: Add Yandex Money icon
Yandex Money supports 2fa, but you andOTP does not have their icon yet.

https://money.yandex.ru/legacy-doc.xml?id=522991&ncrnd=6614
https://money.yandex.ru/i/yandex_dengi_product_icon_cmyk.eps
https://money.yandex.ru/i/yandex_dengi_product_icon_rgb.png

### LINKED ISSUE #441: Add Keeper icon
Please add the logo for Keeper Password Manager

https://www.keepersecurity.com/

### LINKED ISSUE #442: Add pythonanywhere icon
Please add the python anywhere icon.

https://www.pythonanywhere.com/

### LINKED ISSUE #444: Add ORCID icon
Please add the ORCID icon.

https://orcid.org/static/release-1.274.4/img/orcid-logo.svg

### LINKED ISSUE #458: Thumbnail for Bitdefender
Please add a thumbnail for Bitdefender. This service supports TOTP based 2FA.

[Bitdefender.svg](https://www.dropbox.com/s/yzeo0m270ty4a06/Bitdefender.svg)

### LINKED ISSUE #460: Thumbnail for TurboTax
Please add a thumbnail for TurboTax. This service supports TOTP based 2FA.

[TurboTax.svg](https://www.dropbox.com/s/sdex8pnfiqhtrjv/TurboTax.svg)

### LINKED ISSUE #461: Thumbnail for Xing
Please add a thumbnail for Xing. This service supports TOTP based 2FA.

[Xing.svg](https://www.dropbox.com/s/cs344fk38nnrw3p/Xing.svg)

### LINKED ISSUE #462: Thumbnail for Xero
https://en.wikipedia.org/wiki/File:Xero_software_logo.svg

### LINKED ISSUE #463: Add thumbnail for Cisco
https://commons.wikimedia.org/wiki/File:Cisco_logo.svg

### LINKED ISSUE #464: Add thumbnail for Zendesk
https://commons.wikimedia.org/wiki/File:Zendesk_logo.svg

### LINKED ISSUE #465: Add thumbnail for Uphold
https://s3-eu-west-1.amazonaws.com/tpd/logos/567f276c0000ff000586ff46/0x0.png

### LINKED ISSUE #469: [Thumbnail] Add Adafruit
Please considering adding a thumbnail for Adafruit.

**Name**: Adafruit
**Website**: https://www.adafruit.com/
**Link to SVG or EPS image**: https://cdn-shop.adafruit.com/static/adafruit_favicon.svg

### LINKED ISSUE #470: [Thumbnail] Add Hover
Please consider adding a thumbnail for Hover which is domain registrar.

**Name**: Hover
**Website**: https://www.hover.com/
**Link to SVG or EPS image**: https://www.hover.com/packs/src/application/images/common/hv_logo-aeb786424e4c4ba2a199aafaa1960943.svg

### LINKED ISSUE #476: [Thumbnail] Zapier
**Name**: Zapier
**Website**: https://zapier.com
**Link to SVG or EPS image**: https://cdn.zapier.com/zapier/images/logos/zapier-logomark.eps

### LINKED ISSUE #505: [Thumbnail] web.de
Please fill in **all the information below** to help speed up the process!

**Name**: web.de
**Website**: www.web.de
**Link to SVG or EPS image**: ?

### LINKED ISSUE #522: [Thumbnail] Twilio
Please fill in **all the information below** to help speed up the process!

Please considering adding Twilio.

**Name**: Twilio
**Website**: https://www.twilio.com
**Link to SVG or EPS image**: 
[twilio.svg.txt](https://github.com/andOTP/andOTP/files/4485330/twilio.svg.txt)

### LINKED ISSUE #526: [Thumbnail] Etsy
Please fill in **all the information below** to help speed up the process!

Please consider adding Etsy.

**Name**: Etsy
**Website**: https://www.etsy.com
**Link to SVG or EPS image**:
[etsy.svg.txt](https://github.com/andOTP/andOTP/files/4501465/etsy.svg.txt)

### LINKED ISSUE #527: [Thumbnail] Workplace
**Name**: Workplace
**Website**: https://www.workplace.com
**Link to SVG or EPS image**: https://www.adenin.com/assets/images/wp-images/logo/workplace-by-facebook-bot.svg

### LINKED ISSUE #537: [Thumbnail] DreamHost
Please consider adding the DreamHost logo (they're a web hosting provider).

**Name**: DreamHost
**Website**: https://www.dreamhost.com/
**Link to SVG or EPS image**:
[dreamhost-logo.svg.txt](https://github.com/andOTP/andOTP/files/4574718/dreamhost-logo.svg.txt)

### LINKED ISSUE #541: [Thumbnail] Fanatical
Hi. Here is another service that could benefit from a custom icon.

**Name**: Fanatical
**Website**: https://www.fanatical.com/en/
**Link to SVG or EPS image**: [Fanatical logo.zip](https://github.com/andOTP/andOTP/files/4592105/Fanatical.logo.zip) _(extracted from the front-page logo)_

Thank you for your help.

### LINKED ISSUE #542: [Thumbnail] Zoom
Please fill in **all the information below** to help speed up the process!

**Name**: Zoom
**Website**: zoom.us
**Link to SVG or EPS image**: https://zoom.us/brandguidelines

### PR DESCRIPTION
**Pull request by ziegenberg**
Adds icons for JetBrains (Fixes #436), lichess (Fixes #355), Slack (Fixes #419), Bugcrowd (Fixes #411), Fastmail (Fixes #435), Yandex.Money (Fixes #438), Keeper (Fixes #441), PythonAnywhere (Fixes #442), ORCiD (Fixes #444), Bitdefener (Fixes #458), Xing (Fixes #461), Cisco (Fixes #463), Xero (Fixes #462), Xerox, Zendesk (Fixes #464), Zapier (Fixes #476), Church.tools, Atlantic.net, TUWien, Mailchimp, SevDesk, Airbnb (Fixes #429), Bitmex (Fixes #401), TurboTax (Fixes #460), Zoom (Fixes #542), Uphold (Fixes #465), Fanatical (Fixes #541), DreamHost (Fixes #537), Workplace (Fixes #527), Etsy (Fixes #526), Twilio (Fixes #522), Web.de (Fixes #505), Hover (Fixes #470), Adafruit(Fixes #469)

### CODE REVIEW

**Comment by flocke** [comment]
Thanks a lot for your work on the thumbnails. I will check out your PR as soon as I find the time. But as I am starting a new job today so that will probably still be a while.

**Comment by ziegenberg** [comment]
Can be reviewed now.

**Comment by flocke** [comment]
@RichyHBM Could you review this PR if you find the time? I'm planing on publishing v0.7.1 soon and want to include this as well.

**Comment by ziegenberg** [comment]
This closes now 28 open tickets and adds/updates 35 thumbnails.

**Comment by ziegenberg** [comment]
I rebased onto current master and fixed the build: https://travis-ci.org/github/andOTP/andOTP/builds/684510664

**Comment by ziegenberg** [comment]
I think this can be reviewed now. Will it make it into the 0.7.1 release?

**Comment by flocke** [comment]
I am planing to include it, but it all depends on how much time Richy or I have to review it before.

File: app/src/main/res/drawable/thumb_hover.xml [hunk 1]
```diff
@@ -0,0 +1,10 @@
...
+      android:strokeWidth="1.72"
+      android:fillColor="#ffffff"/>
+</vector>
```
Comment (flocke) [inline app/src/main/res/drawable/thumb_hover.xml:10]: This logo is badly visible on the light background, we should probably better use this one: https://upload.wikimedia.org/wikipedia/commons/4/44/Hover_logo.svg

Comment (ziegenberg) [reply app/src/main/res/drawable/thumb_hover.xml:10]: will fix in an instant

Comment (ziegenberg) [reply app/src/main/res/drawable/thumb_hover.xml:94]: I broke the one path used for the whole word into several paths for each single letter. Android Studio complains if a path is longer than 800 characters and now its less

**Comment by ziegenberg** [comment]
shall I rebase onto master for merging?

**Comment by flocke** [comment]
> shall I rebase onto master for merging?

No need to, it can be merged without problems.

**Comment by flocke** [comment]
Thank you for all your work on the thumbnails, this actually helps me a lot!

**Pull request merged**